inst/scripts/run_sctour.py

```diff
@@ -0,0 +1,53 @@
+from pathlib import Path
+import sctour as sct
+import scanpy as sc
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import json
+
+def run_sctour(GEXfile, metafile, exclusion_json_path, ptime_out_file):
+    adataObj = sc.read_10x_h5(GEXfile)
+    info = pd.read_csv(metafile, sep=',', index_col=0)
+    with open(exclusion_json_path) as f:
```

Comment:
it seems like it is plausible for the user to not supply an exclusion list, so i would make sure the code works OK if nothing is provided. Or perhaps do a "None" check and skip this part if exclusion_json_path==None?